build/tsconfig.commonjs.json

```diff
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "module": "commonjs"
   },
-  "include": ["../packages/src", "../packages/types", "../types"]
```

Comment:
Path `packages/types` is not existing, `packages/src` should be `packages/react/src`